src/utils/gradeColor.ts

```diff
@@ -10,5 +10,8 @@ exports.getGradeColorByCategoryName = (name) => {
       return '20A4F7';
     case name.includes('zielony'):
       return '6ECD07';
+    default:
+      // Return a default color if no match is found
+      return 'FFFFFF'; // or any other default color
```

Comment:
```suggestion
      return 'FFFFFF';
```

Reply:
There wasn't any default color earlier and the compiler was giving errors to me so I added one myself
Is white alright? I will get on with the changes.

Reply:
I don't have problem with that, I just think your comment is something like:
![image](https://github.com/user-attachments/assets/393fa94e-d56a-4e41-9927-e8258cf77945)

Reply:
Ah my bad, lol.